source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx

```diff
@@ -561,21 +561,13 @@ async function readInlangConfig(args: {
 			$fs: args.fs.promises,
 		}
 		const file = await args.fs.promises.readFile("./inlang.config.js", "utf-8")
-		const withMimeType = "data:application/javascript;base64," + btoa(file.toString())
+		const withMimeType =
+			"data:application/javascript;base64," + Buffer.from(file).toString("base64")
 
 		const module = await import(/* @vite-ignore */ withMimeType)
-		// account for breaking change from renaming the config
-		// https://github.com/inlang/inlang/issues/291
```

Comment:
issue was closed. So we can remove this legacy code

Reply:
Yes